vue-frontend/src/components/jobs/PerksAndBenifits.vue

```diff
@@ -84,6 +88,7 @@ import celebration from "@/assets/images/benifits/jobs_canopas_celebration.svg";
 export default {
   data() {
     return {
+      event: "",
```

Comment:
Remove this.
Once any comment is added, then check for all occurrences. It's not about just that code only.